src/seedu/addressbook/commands/ViewAllCommand.java

```diff
@@ -24,15 +25,25 @@ public ViewAllCommand(int targetVisibleIndex) {
         super(targetVisibleIndex);
     }
 
-
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        String output = "";
+        for(Printable E: printables) {
```

Comment:
`for` should be followed by a whitespace. Refer to our Java coding standard.
Also, Java has a `StringJoiner` class that should be useful for joining strings.
Ideally, this `getPrintableString` should be used somewhere. For example in the file [ReadOnlyPerson.java](https://github.com/nus-cs2103-AY1718S1/addressbook-level3/blob/master/src/seedu/addressbook/data/person/ReadOnlyPerson.java#L71) you can replace `getPhone()` with `getPrintableString(getPhone())`